x/posts/keeper/common_test.go

```diff
@@ -42,17 +34,15 @@ type KeeperTestsuite struct {
 	storeKey sdk.StoreKey
 	k        keeper.Keeper
 
-	ak profileskeeper.Keeper
+	ak *testutil.MockProfilesKeeper
 	sk subspaceskeeper.Keeper
```

Comment:
`x/posts` highly relies on `x/subspaces`, so `testutil.MockSubspacesKeeper` cannot be used as a replacement because it requires the use of the non-pure function `IterateSubspaces` within the expected subspaces keeper.